operator/cmd/mesh/manifest-migrate.go

```diff
@@ -105,12 +108,23 @@ func translateFunc(values []byte, l *Logger) error {
 	}
 
 	// Not IstioControlPlane, try Helm values.yaml.
-	ts, err := translate.NewReverseTranslator(binversion.OperatorBinaryVersion.MinorVersion)
+	mvs := binversion.OperatorBinaryVersion.MinorVersion
```

Comment:
to maintain the same abstraction level in the function, i recommend creating a helper to test whether the input is  values (i have that test function for ICP per comment above but it would be good to have something similar for helm).